src/en/discover/case-studies/us-signal-expands-storage/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "US Signal Expands Distributed Storage Infrastructure Capabilities"
+image: "images/photo-aquarium-silhouette.jpg"
+sponsor:
+  name: SoftIron
+  logo: "/assets/bitmaps/logo-softiron.png"
+  website: "https://www.softiron.com"
+tags:
+  - information
+  - technology
+  - msp
+---
+
+## US Signal scales its Storage As A Service (STaaS) offering to help meet growing demand
+
+Founded in 2001, US Signal is the largest debt-free, privately held, data center services provider in the Midwest, delivering industry-leading data center technologies, cloud solutions, data protection and managed services to help companies transform their IT operations and meet ever-changing technology and business needs.
+
+They recently increased their storage capacity by over a petabyte with a no-downtime migration across multiple distributed data center sites in four midwest states, including Illinois, Indiana, Michigan, and Wisconsin.
+
+# Increasing customer demand required capacity to scale-out rapidly
```

Comment:
@fishoutoforder Ideally, we wouldn't use a h1 after a h2 due to semantics.

Could this be a h2 ?